privcount/tally_server.py

```diff
@@ -143,6 +144,29 @@ def refresh_config(self):
             else:
                 ts_conf['counters'] = conf['counters']
 
+            # Load the sigma values from the data collector configuration
+            # this is a cut-down version of the DC loading script
+            dc_conf = conf['data_collector']
```

Comment:
In practice, we have been separating the config files for the DC, SK, and TS, to simplify the config associated with running each node type. So, the tally server's config file would not have a 'data_collector' section available.

We could

  1. add the counter sigmas file as required input for the tally server
  1. have the data collectors send the sigmas back to the tally server along with the counts that it currently sends

I would normally go with 2. OTOH, eventually the sigmas file will go away and be replaced with auto sigma generation, and so this will all change anyway. So maybe we should keep it simple for now and go with 1. Thoughts?